crates/modelardb_server/src/manager.rs

```diff
@@ -117,28 +111,24 @@ impl Manager {
         Ok(())
     }
 
-    /// If `action_type` is `CreateTable` or `KillNode`, check that the request actually
-    /// came from the manager. If the request is valid, return [`Ok`], otherwise return
-    /// [`ModelarDbError`].
-    pub fn validate_action_request(
-        &self,
-        action_type: &str,
-        metadata: &MetadataMap,
-    ) -> Result<(), ModelarDbError> {
+    /// If `action_type` is `CreateTable` or `KillNode`, check that the request actually came from
```

Comment:
This is not relevant to this PR but i forgot to add `DropTable` to the action types listed in the doc comment so maybe just add it now.

Reply:
Fixed.